### DISCUSSION TITLE
Prev/Next Question Experience

### DISCUSSION THREAD

**Discussion by gfeeng** [greatfrontend/greatfrontend#57] [Ideas/Feature Requests]
> At the bottom of a question's description, in "Try these questions next" and "Similar Questions" sections, it would be great if the completed questions are indicated differently than the completed ones. A completed question could e.g. have a green checkmark like the one appearing at the start of a question in https://www.greatfrontend.com/questions

> tiny UX suggestion - next/prev problem buttons. would be very useful when you get into a "flow"

> Is it possible to navigate to different/next questions and sections within a study plan (while in the code workspace/editor view)? Without having to select "Study Plans" from navbar again. I think right now, there is an option to go to next questions, but they are not within the same context of the selected study plan. Thanks!

> It would be great if I could use filters on questions in a study plan. Filters now seem to appear only on "See all" questions pages.

> add `prev` and `next` buttons on the question's user-interface page 😭 it is annoying to go back and pick a question 

> At the bottom of a question's description, in "Try these questions next" and "Similar Questions" sections, it would be great if the completed questions are indicated differently than the completed ones. A completed question could e.g. have a green checkmark like the one appearing at the start of a question in

**Comment by yangshun** [comment]
This has been shipped.
